File: C2002LDoiCan/APC#/22-06-2021/exam1/models/Person.cs
using System;
namespace exam1.models
{
    public class Person
    {
        private String _IDCard;
        private String _name;
        private int _age;
        //read-only => ko khoi tao kieu Builder Pattern duoc
        public String IDCard { get; }
        public String Name { get; }
        public String Age { get; }
        public Person(String IDCard, String name, int age) {
            _IDCard = IDCard;
            _name = name;
            _age = age;

        }
        public override string ToString()
        {
            return $"id's card : {_IDCard}\n" +
                    $"name : {_name}\n" +
                    $"age: {_age}\n";
        }
    }
}

File: C2002LDoiCan/ADF1/28-01-2021/src/com/models/Person.java
package com.models;

public class Person {
    String name = "";
    Integer age = 18;
    String email = "";
    final String hairColor = "black";
    /*
    {
        //initialization block
        name = "";
        age = 18;
        email = "";
        if(name == null) {

        }
        System.out.println("Day la khoi khoi tao");
    }

     */
    public Person() {
        System.out.println("This is constructor");
    }
}
